sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -345,6 +346,46 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
   }
 
   /**
+   * Loads a CSV file and returns the result as a [[DataFrame]].
+   *
+   * This function goes through the input once to determine the input schema. If you know the
+   * schema in advance, use the version that specifies the schema to avoid the extra scan.
```

Comment:
"If you know the schema in advance, use the version that specifies the schema to avoid the extra scan."

There is no version that does that. Maybe just say "To avoid going through the entire data once, specify the schema explicitly using [[schema]]."